recipe/entry_point.py

```diff
@@ -12,6 +13,8 @@ def map(self, func, *iterables):
                 yield func(thing)
 
 if __name__ == '__main__':
+    freeze_support()
```

Comment:
This could be moved into tighter scope, but I think having it here is fine in case any future operations also use the parallel processing.  Better to catch them all than leave people scratching their heads later.